checkbox-provider-ce-oem/units/socketcan/manifest.pxu

```diff
@@ -0,0 +1,5 @@
+unit: manifest entry
+id: socket_can_fd
+_prompt: Is the CAN FD capable device:
+_name: CAN FD capable devices
+value-type: bool
```

Comment:
- Please leave a blank at the end of file.
- Please rename the `socket_can_fd` to be `has_socket_can_fd`

Reply:
thanks for the comments, fixed in latest changes.